ncm-nss/src/test/perl/configure.t

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl
+# -*- mode: cperl -*-
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Quattor qw(simple);
+use NCM::Component::sysctl;
```

Comment:
and because you don't do anything with mock below, you can maybe remove those and use the correct one here?